src/main/CEO.java

```diff
@@ -0,0 +1,35 @@
+
+
+import java.util.*;
+
+public class CEO {
+    private String name;
+
+    public String getName() {
+        return name;
+    }
+
+    public void setName(String name) {
```

Comment:
- Settery i Gettery umieszczamy na końcu klasy
- I generujemy je dopiero jak są nam potrzebne nie ma sensu robić ich do wszystkiego bo rozhermetyzowywuje nam to klaskę:)